src/components/CopyToClipboard/CopyToClipboardTs.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, Vue, Prop } from 'vue-property-decorator'
+import { UIHelpers } from '@/core/utils/UIHelpers'
+import { NotificationType } from '@/core/utils/NotificationType'
+
+
+@Component
+export class CopyToClipboardTs extends Vue {
+  @Prop({ default: null }) prepareCopyData: any
+  copyToClipboard() {
+    let prepareCopyData: any = this.prepareCopyData
+    if((Array.isArray(prepareCopyData) && prepareCopyData.length == 0) || !prepareCopyData){
+      return
+    }
+    if (Array.isArray(prepareCopyData)) {
+      prepareCopyData = (prepareCopyData as Array<any>).join(' ')
+    }
```

Comment:
this should not **always** be done for "copy to clipboard" ; this is _specific to mnemonic words phrases_.

Please extract the join + adding spaces to the MnemonicInput component.